src/style.css

```diff
@@ -1,3 +1,69 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-size: 16px;
+
+  --nav-font-color: #000;
+  --primary-color: #00acdc;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  background-color: #fff;
+}
+
+.logo__image {
+  display: block;
+}
+
+.nav__list {
+  display: flex;
+  margin: 0;
+  padding: 0;
+  list-style: none;
+}
+
+.nav__item:not(:first-child) {
+  margin-left: 20px;
+}
+
+.nav__link {
+  display: flex;
+  text-transform: uppercase;
+  align-items: center;
+  height: 60px;
+  font-weight: 500;
+  text-decoration: none;
+  color: var(--nav-font-color);
+  position: relative;
+  font-size: 0.75rem;
+}
+
+.nav__link::after {
```

Comment:
```suggestion
.is-active::after {
```

Reply:
If I do this, my transition won't work correctly, won't it?